backend/de.metas.postfinance/src/main/sql/postgresql/system/90-postfinance/5717340_sys_gh17320_Download_CustomerRegistrationMessage_Scheduler.sql

```diff
@@ -0,0 +1,3 @@
+-- 2024-02-14T17:38:02.173Z
+INSERT INTO AD_Scheduler (AD_Client_ID,AD_Org_ID,AD_Process_ID,AD_Role_ID,AD_Scheduler_ID,Created,CreatedBy,CronPattern,Description,EntityType,Frequency,FrequencyType,IsActive,IsIgnoreProcessingTime,KeepLogDays,ManageScheduler,Name,Processing,SchedulerProcessType,ScheduleType,Status,Supervisor_ID,Updated,UpdatedBy) VALUES (1000000,1000000,585355,0,550111,TO_TIMESTAMP('2024-02-14 19:38:01.9','YYYY-MM-DD HH24:MI:SS.US'),100,'0 6 * * *','Download Customer Registration Messages from PostFinance every night at 6','D',0,'D','Y','N',7,'N','PostFinance_Download_Customer_Registration_Message','N','P','C','NEW',100,TO_TIMESTAMP('2024-02-14 19:38:01.9','YYYY-MM-DD HH24:MI:SS.US'),100)
+;
```

Comment:
I think the scheduler shouldn't be active by default, to avoid Errors because PostFinance isn't configured for an customer

Reply:
good point, thx. updated

Reply:
Added mig file in private repo to activate it